auto_round/inference/backend.py

```diff
@@ -205,6 +240,34 @@ def fp8_static_scheme_checker(
     requirements=["auto-round>0.6.0"],
 )
 
+# MXFP8
+
+BackendInfos["auto_round:mxfp8"] = BackendInfo(
+    device=["xpu", "cuda", "cpu"],
+    packing_format="",
+    sym=[True],
+    dtype=["float32", "float16", "bfloat16"],
+    bits=[8],
+    priority=0,
+    checkers=[mxfp8_scheme_checker],
```

Comment:
when will you fix the TODO in your last pr, that to expand the weight only scheme to activation instead of using a function